apis/pipeline/pipeline.yaml

```diff
@@ -2662,6 +2662,31 @@ paths:
                 default:
                     $ref: '#/components/responses/Error'
 
+    /api/v1/orgs/{orgId}/clusters/{id}/update:
```

Comment:
**Note**: I haven't made up my mind yet whether having a `/api/v1/orgs/{orgId}/clusters/{id}` endpoint for node pool update (and potentially other cluster level, under-cluster resource updates) and  `/api/v1/orgs/{orgId}/clusters/{id}/update` for cluster version update is clear/intuitive enough from user perspective, going to think on it more, I don't have any alternative suggestions for now.

(I think the distinction between the cluster master node changes and everything else "under" the cluster such as other nodes *can* be communicated and understood well enough, it's only the endpoint paths I'm still chewing on, but for now we are kind of locked in with the latter due to backward compatibility reasons anyway and I have nothing against the new endpoint path.)